lib/modules/manager/vendir/extract.ts

```diff
@@ -71,12 +91,22 @@ export function extractPackageFile(
   // grab the helm charts
   const contents = pkg.directories.flatMap((directory) => directory.contents);
   for (const content of contents) {
-    const dep = extractHelmChart(content.helmChart, config.registryAliases);
-    if (dep) {
-      deps.push({
-        ...dep,
-        depType: 'HelmChart',
-      });
+    if ('helmChart' in content && content.helmChart) {
+      const dep = extractHelmChart(content.helmChart, config.registryAliases);
+      if (dep) {
+        deps.push({
+          ...dep,
+          depType: 'HelmChart',
```

Comment:
should we move this to the extract functions? So we don't need to clone opjects here again? (perf)

Reply:
Should we? No idea. But I did if it speeds up performance. I don't think it will matter much, in this case, because for vendir the blocking thing will always be vendir sync, but happy to help where possible.